notebooks/official/tabular_workflows/wide_and_deep_on_vertex_pipelines.ipynb

```diff
@@ -170,7 +170,7 @@
       "source": [
```

Comment:
<p>Vertex SDK -&gt; Vertex AI SDK</p><p> </p><p>See the other notebook for suggested text improvements</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1378/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
Done. Addressed similar issue in the Wide & Deep notebook.